sql-cloudant/src/main/scala/org/apache/bahir/cloudant/DefaultSource.scala

```diff
@@ -125,7 +125,7 @@ class DefaultSource extends RelationProvider
           /* Create a streaming context to handle transforming docs in
           * larger databases into Spark datasets
           */
-          val ssc = new StreamingContext(sqlContext.sparkContext, Seconds(10))
+          val ssc = new StreamingContext(sqlContext.sparkContext, Seconds(8))
```

Comment:
I'm amazed this parameter was hard-coded as it seems to be a fairly critical tuning parameter for streaming. I guess this is one for another PR.

Reply:
Yes, this was not visible as part of the initial internal `_changes` work.  I will create a new JIRA item and PR for this.